database/src/main/resources/changesets/call_caching_allow_result_reuse_fix.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <property name="boolean.type" value="BOOLEAN" dbms="!mysql"/>
+    <property name="boolean.type" value="TINYINT" dbms="mysql"/>
+
+    <changeSet id="call_caching_fix_boolean" author="mcovarr">
+        <modifyDataType columnName="ALLOW_RESULT_REUSE"
+                        newDataType="${boolean.type}"
+                        tableName="CALL_CACHING_RESULT_METAINFO"/>
+    </changeSet>
+
+    <changeSet id="call_caching" author="mcovarr">
+        <addDefaultValue columnName="ALLOW_RESULT_REUSE"
+                         defaultValueBoolean="true"
```

Comment:
I'm not 100% sure.  The datatype that was there before didn't format nicely in the mysql client, but I think the Liquibase / Slick consistency checker would have complained if the Liquibase was defaulting but the Slick was not.